# Goal
string object_name

---
# Result
bool success
string message

---
# Feedback
float32 progress
